addons/acodec/acodec.h

```diff
@@ -49,4 +58,9 @@ ALLEGRO_AUDIO_STREAM *_al_load_ogg_vorbis_audio_stream_f(ALLEGRO_FILE* file,
    size_t buffer_count, unsigned int samples);
 #endif
 
+#ifdef ALLEGRO_CFG_ACODEC_SNDFILE
```

Comment:
These should not be in this pull request, yet another crumble of the previous implementation. Sorry ^__^